<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Admin Version&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>100</y>
     <width>1001</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="users_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="review_list">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dowload_users_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>Upload users data base</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dowload_reviews_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <property name="text">
    <string>Upload review data bse</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>520</y>
     <width>981</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>482</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Quit from application</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
